src/libraries/System.Text.RegularExpressions/gen/Resources/Strings.resx

```diff
@@ -242,7 +230,7 @@
     <value>This platform does not support writing compiled regular expressions to an assembly.</value>
   </data>
   <data name="QuantifierAfterNothing" xml:space="preserve">
-    <value>Quantifier {x,y} following nothing.</value>
+    <value>Quantifier '{0}' following nothing.</value>
```

Comment:
what is {0} replaced with here? I would expect ideally something like `{3}` or `{5,7}` but perhaps possibly `+` etc?